Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -2268,7 +2268,7 @@ final class BuildPlanTests: XCTestCase {
         result.checkProductsCount(1)
         result.checkTargetsCount(4)
         
-        XCTAssertTrue(result.targetMap.values.contains { $0.target.name == "Utils" && $0.target.moduleAliases?["Logging"] == "FooLogging" })
+        XCTAssertTrue(result.targetMap.values.contains { $0.target.name == "Utils" && $0.target.moduleAliases == nil })
```

Comment:
has this test changes because the behavior is now more correct than before?

Reply:
correct